docs/upgrade-to-v5.md

```diff
@@ -50,3 +50,18 @@ TODO(NODE-4771): serializeFunctions bug fix makes function names outside the asc
 We have set our typescript compilation target to `es2020` which aligns with our minimum supported Node.js version 14+. The following is from the typescript release notes on es2020 support, so it's some of the syntax that can be expected to be preserved after compilation:
 
 > This will preserve newer ECMAScript 2020 features like optional chaining, nullish coalescing, export * as ns, and dynamic import(...) syntax. It also means bigint literals now have a stable target below esnext.
+
+### Remove deprecated ObjectId methods
+
+The following deprecated methods have been removed:
+
+- `ObjectId.prototype.generate`
+  - Instead, generate a new ObjectId with the constructor: `new ObjectId()`
```

Comment:
in the ticket description, we mention the static generate, and it's also the suggestion in the corresponding deprecation message

Reply:
updated